### PR TITLE
fix: flaky test `Send NFT should not be able to send ERC1155 NFT with invalid amount`

### LINKED ISSUE #25288: flaky test: `Send NFT should not be able to send ERC1155 NFT with invalid amount`
### What is this about?

- ci job https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/87308/workflows/69d0b371-2e2a-4aed-b6ef-789f0f3dd7f1/jobs/3194516/artifacts
- screenshot 

image

### Scenario

_No response_

### Design

_No response_

### Technical Details

_No response_

### Threat Modeling Framework

_No response_

### Acceptance Criteria

_No response_

### Stakeholder review needed before the work gets merged

- [ ] Engineering (needed in most cases)
- [ ] Design
- [ ] Product
- [ ] QA (automation tests are required to pass before merging PRs but not all changes are covered by automation tests - please review if QA is needed beyond automation tests)
- [ ] Security
- [ ] Legal
- [ ] Marketing
- [ ] Management (please specify)
- [ ] Other (please specify)

### References

_No response_

### PR DESCRIPTION
**Pull request by seaona**
## **Description**
This PR fixes the flaky test `Send NFT should not be able to send ERC1155 NFT with invalid amount`.

The error is `ElementNotInteractableError: Element <button class="mm-box nft-item__container"> could not be scrolled into view`.

After looking into the code, we see that there is actually no reason why we are trying to scroll to the button before clicking it. This PR removes this step as well as other unnecessary steps and waits alongside the spec.

:racehorse:  It now runs faster (notice we were adding up more than **15seconds** of unnecessary delays) and without flakiness.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25289?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/25288

## **Manual testing steps**

1. Run test multiple times `ENABLE_MV3=false FIREFOX_SNAP=false yarn test:e2e:single test/e2e/tests/tokens/nft/send-nft.spec.js --browser=firefox --leave-running --retryUntilFailure --retries=10`
2. Check ci

## **Screenshots/Recordings**

https://github.com/MetaMask/metamask-extension/assets/54408225/9cee8b81-7d1f-4f38-9656-4f33e62d37c2

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: test/e2e/tests/tokens/nft/send-nft.spec.js [hunk 1]
```diff
@@ -173,21 +165,16 @@ describe('Send NFT', function () {
...
 
-        const erc1155Token = await driver.findElement('.nft-item__container');
-        await driver.scrollToElement(erc1155Token);
-        await driver.delay(1000);
-        await driver.clickElement('.nft-item__container');
+        await driver.clickElement('[data-testid="nft-network-badge"]');
 
         await driver.clickElement({ text: 'Send', tag: 'button' });
 
-        await driver.delay(10000);
```
Comment (seaona) [inline test/e2e/tests/tokens/nft/send-nft.spec.js:189]: :exploding_head:

File: test/e2e/tests/tokens/nft/send-nft.spec.js [hunk 2]
```diff
@@ -93,16 +94,13 @@ describe('Send NFT', function () {
...
+      async ({ driver, ganacheServer }) => {
+        await logInWithBalanceValidation(driver, ganacheServer);
 
         // Fill the send NFT form and confirm the transaction
         await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
 
-        await driver.delay(1000);
-        const erc1155Token = await driver.findElement('.nft-item__container');
-        await driver.scrollToElement(erc1155Token);
-        await driver.delay(1000);
```
Comment (seaona) [inline test/e2e/tests/tokens/nft/send-nft.spec.js:105]: 1000

File: test/e2e/tests/tokens/nft/send-nft.spec.js [hunk 3]
```diff
@@ -93,16 +94,13 @@ describe('Send NFT', function () {
...
       },
-      async ({ driver }) => {
-        await unlockWallet(driver);
+      async ({ driver, ganacheServer }) => {
+        await logInWithBalanceValidation(driver, ganacheServer);
 
         // Fill the send NFT form and confirm the transaction
         await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
 
-        await driver.delay(1000);
```
Comment (seaona) [inline test/e2e/tests/tokens/nft/send-nft.spec.js:102]: 1000s

File: test/e2e/tests/tokens/nft/send-nft.spec.js [hunk 4]
```diff
@@ -112,7 +110,6 @@ describe('Send NFT', function () {
           'input[placeholder="Enter public address (0x) or ENS name"]',
           '0xc427D562164062a23a5cFf596A4a3208e72Acd28',
         );
-        await driver.delay(1000);
```
Comment (seaona) [inline test/e2e/tests/tokens/nft/send-nft.spec.js:115]: 3000s

File: test/e2e/tests/tokens/nft/send-nft.spec.js [hunk 5]
```diff
@@ -148,11 +145,6 @@ describe('Send NFT', function () {
         // Go back to NFTs tab and check the imported NFT is shown as previously owned
         await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
 
-        const refreshList = await driver.findElement(
-          '[data-testid="refresh-list-button"]',
-        );
-        await driver.scrollToElement(refreshList);
-        await driver.delay(1000);
```
Comment (seaona) [inline test/e2e/tests/tokens/nft/send-nft.spec.js:155]: 4000s

File: test/e2e/tests/tokens/nft/send-nft.spec.js [hunk 6]
```diff
@@ -173,21 +165,16 @@ describe('Send NFT', function () {
...
-        await unlockWallet(driver);
+      async ({ driver, ganacheServer }) => {
+        await logInWithBalanceValidation(driver, ganacheServer);
 
         // Fill the send NFT form and confirm the transaction
         await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
 
-        const erc1155Token = await driver.findElement('.nft-item__container');
-        await driver.scrollToElement(erc1155Token);
-        await driver.delay(1000);
```
Comment (seaona) [inline test/e2e/tests/tokens/nft/send-nft.spec.js:184]: 5000s

**Review by chloeYue — APPROVED** [review]
LGTM !

**Review by DDDDDanica — APPROVED** [review]

**Comment by DDDDDanica** [comment]
👍🏻👍🏻👍🏻

**Pull request merged**